docs/source/tools.rst

```diff
@@ -0,0 +1,73 @@
+
+Tools Guide
+=================
+
+First, make sure that PyU4V is `installed <http://pyu4v.readthedocs.io/en/latest/installation.html>`_.
+Then visit the `quick_start <http://pyu4v.readthedocs.io/en/latest/quick_start.html>`_ section to make
+sure you have secure connectivity to your array.
+
+OpenStack
+---------
+
+**Description**
+
+This script facilitates the seamless(live) migration of volumes from the SMIS
+masking view structure to the REST masking view structure introduced in Pike.
+This is only applicable if you have existing volumes created in Ocata or an earlier
+release.
+
+.. important::
+   - Running this script is not necessary unless you intend 'Live Migrating' from one compute node to another.
```

Comment:
These lines need wrapped at 79 chars where possible, there are some tables elsewhere that are impossible to get into that size but they are the only exception in the .rst files.

Reply:
sure